### PR TITLE
Created OONI FAQ

### PR DESCRIPTION
**Pull request by agrabeli**

### CODE REVIEW

File: content/about/faq.md [hunk 1]
```diff
@@ -0,0 +1,1553 @@
...
+
+    * [Can I measure internet censorship remotely in another country?](#can-i-measure-internet-censorship-remotely-in-another-country)
+
+    * [What types of data does OONI Probe collect?](#what-types-of-data-does-ooni-probe-collect)
+
+    * [How can I opt-out from sending OONI measurements?](#how-can-i-opt-out-from-sending-ooni-measurements)
+
+    * [What are the risks of running OONI Probe?](#what-are-the-risks-of-running-ooni-probe)
+
+    * [How can I reduce these risks?](#how-can-i-reduce-these-risks)
```
[the remark this answers is not in the document]
Comment (agrabeli) [reply content/about/faq.md]: Good point! I have edited this to make it an explicit, stand-alone question, per your suggestion.

File: content/about/faq.md [hunk 2]
```diff
@@ -0,0 +1,1553 @@
...
+    * [Why doesn’t my country have any recent measurements on OONI Explorer?](#why-doesnt-my-country-have-any-recent-measurements-on-ooni-explorer)
+
+
+## About OONI
+
+### What is OONI?
+
+The [Open Observatory of Network Interference (OONI)](https://ooni.org/) is a free software project that aims to
+empower decentralized efforts in increasing transparency of internet
+censorship (and other forms of network interference) around the world.
```
[the remark this answers is not in the document]
Comment (agrabeli) [reply content/about/faq.md]: I have edited this to reduce confusion.

File: content/about/faq.md [hunk 3]
```diff
@@ -0,0 +1,1553 @@
...
+
+### What does OONI do?
+
+To empower decentralized efforts in increasing transparency of internet
+censorship worldwide, we:
+
+* **Build free and open source software for detecting
+internet censorship.** [OONI Probe](https://ooni.org/install/)
+is a network measurement tool that you can run to measure internet
+censorship and other forms of network interference.
```
[the remark this answers is not in the document]
Comment (agrabeli) [reply content/about/faq.md]: I have re-phrased this to say that OONI Probe measures various forms of internet censorship, as well as network speed and performance.

File: content/about/faq.md [hunk 4]
```diff
@@ -0,0 +1,1553 @@
...
+
+* **Analyze and openly publish network measurements.** Every day, we
+analyze and openly publish measurements on [OONI Explorer](https://explorer.ooni.org/), a web interface hosting
```
[the remark this answers is not in the document]
Comment (agrabeli) [reply content/about/faq.md]: That's a great suggestion -- I edited it to say "website" instead.

File: content/about/faq.md [hunk 5]
```diff
@@ -0,0 +1,1553 @@
...
+affects services that we commonly use, or when their blocking
+receives media coverage. The [blocking of Telegram and Instagram in Iran](https://ooni.org/post/2018-iran-protests/), for example,
+received a fair amount of [media coverage](http://www.bbc.com/news/world-middle-east-42529576) and
+international attention, whereas the [blocking of religious and ethnic minority sites](https://ooni.org/post/iran-internet-censorship/#human-rights-issues)
+may be less known.
+
+2. **Internet censorship often differs from network to network within
+a country.** In many countries around the world, Internet Service
+Providers (ISPs) block access to different websites at their own
+discretion (often as a result of vague government orders). A site
```
[the remark this answers is not in the document]
Comment (agrabeli) [reply content/about/faq.md]: That's a good point. I have edited this section.

File: content/about/faq.md [hunk 6]
```diff
@@ -0,0 +1,1553 @@
...
+harder to determine whether a site is intentionally blocked, or if
+it’s inaccessible due to other reasons (such as a transient
+network failure or a [case of DNS misconfiguration](https://ooni.org/post/not-quite-network-censorship/)).
+
+4. **Some ISPs adopt a combination of different
+censorship techniques.** While your ISP may serve a block page for
+some sites (informing you that those sites are intentionally
+blocked), they may block different sites with different techniques
+-- limiting your ability to notice the blocking of those
+other sites. In some countries, ISPs even [block the same sites with different techniques](https://ooni.org/post/iran-internet-censorship/#blocked-domains-and-tools).
```
[the remark this answers is not in the document]
Comment (agrabeli) [reply content/about/faq.md]: I have seen this in some countries through OONI data, but not sure if we can say "many" instead.

File: content/about/faq.md [hunk 7]
```diff
@@ -0,0 +1,1553 @@
...
+blocking all IP addresses originating from the country that you
+are trying to access it from (in compliance with laws
+and regulations).
+
+Measuring internet censorship is, therefore, important because it allows
+us to **inspect a network** and to examine the reasons as to why and how
+we are (or aren’t) able to connect to an internet service.
+
+When inspecting a network we observe, for example, that our ISP is
+spoofing the IP address of the website that we’re trying to access, this
```
[the remark this answers is not in the document]
Comment (agrabeli) [reply content/about/faq.md]: Great suggestion! DNS spoofing is one of the terms in the OONI glossary. We can link to it (and other terms) once the glossary is published.

File: content/about/faq.md [hunk 8]
```diff
@@ -0,0 +1,1553 @@
...
+
+### What is OONI’s relationship with The Tor Project?
+
+OONI was born out of [The Tor Project](https://www.torproject.org/).
+
+Back in 2011, Tor Project developers started creating open methodologies
+and frameworks with the goal of measuring various forms of network
+interference, resulting in the creation of OONI.
+
+Ever since, OONI has been [one of the projects hosted under The Tor Project Inc.](https://2019.www.torproject.org/projects/projects.html.en), which
```
[the remark this answers is not in the document]
Comment (agrabeli) [reply content/about/faq.md]: I was also wondering the same question...

File: content/about/faq.md [hunk 9]
```diff
@@ -0,0 +1,1553 @@
...
+has served as OONI’s fiscal sponsor.
+
+### How can I get involved?
+
+Depending on your skills and interests, there are many ways you can get
```
[the remark this answers is not in the document]
Comment (agrabeli) [reply content/about/faq.md]: Great call! Edited this per your suggestion.

File: content/about/faq.md [hunk 10]
```diff
@@ -0,0 +1,1553 @@
...
+The censorship findings are only as interesting as the websites
+that are tested. Please help us improve the quality of
+measurements by adding websites that you think should be tested!
+
+* **[Translate the OONI Probe apps](https://wiki.localizationlab.org/index.php/OONI_Probe)**.
+By [translating the OONI Probe apps](https://www.transifex.com/otf/ooniprobe/), you are enabling
+more communities to measure internet censorship.
+
+* **Participate in community discussions.** Join us on
+[Slack](https://slack.ooni.org/) or IRC (ircs://irc.oftc.net:6697/#ooni) for real-time discussions,
```
[the remark this answers is not in the document]
Comment (agrabeli) [reply content/about/faq.md]: Yep! Edited accordingly.

**Comment by agrabeli** [comment]
Thank you @bassosimone for the review & feedback!

**Comment by agrabeli** [comment]
@hellais please review this carefully.

File: content/about/faq.md [hunk 11]
```diff
@@ -0,0 +1,1550 @@
...
+
+* Blocking of censorship circumvention tools (such as Tor and Tor
+bridges);
+
+* Presence of middleboxes (i.e. systems that could be responsible for
+censorship or surveillance);
+
+* Speed and performance of networks;
+
+* Video streaming performance.
```
Comment (hellais) [inline content/about/faq.md]: Maybe this is redundant and can be assimilable to the previous point?

File: content/about/faq.md [hunk 12]
```diff
@@ -0,0 +1,1550 @@
...
+* Windows & macOS: https://github.com/ooni/probe-desktop/releases (new and experimental)
+
+* Linux: https://github.com/ooni/probe-cli/releases (new and experimental)
+
+Older versions of OONI Probe (which are no longer updated) are available
+for:
+
+* Linux & macOS: https://ooni.torproject.org/install/ooniprobe
+
+* Raspberry Pi: https://ooni.torproject.org/install/lepidopter/
```
Comment (hellais) [inline content/about/faq.md]: These links should point to `ooni.org`

File: content/about/faq.md [hunk 13]
```diff
@@ -0,0 +1,1550 @@
...
+user attempts to access a website they are not permitted to view.
+If an ISP blocks access to a website using *different censorship
+techniques* (which is often the case), those cases will *not* show
+up under the “Confirmed” option on OONI Explorer.
+
+* **We (the OONI team) have detected that block page and added its
+fingerprint to the OONI database (and updated OONI Explorer).**
+There are probably many block pages out there that we haven’t
+detected yet. If you come across block pages that aren’t flagged
+on OONI Explorer, please [share them with us](https://github.com/ooni/pipeline/blob/master/af/oometa/003-fingerprints.install.sql).
```
Comment (hellais) [inline content/about/faq.md]: This should rather point to the ooni/pipeline issue repository with the new issue marked, like this: https://github.com/ooni/pipeline/issues/new

File: content/about/faq.md (hunk 11 above)
Comment (agrabeli) [reply content/about/faq.md]: Sure (I was thinking of the DASH test). I have deleted it.

File: content/about/faq.md (hunk 12 above)
Comment (agrabeli) [reply content/about/faq.md]: Good catch! Edited.

Comment (agrabeli) [reply content/about/faq.md]: I also searched for all the other cases where we have ooni.torproject.org and replaced them with ooni.org.

File: content/about/faq.md (hunk 13 above)
Comment (agrabeli) [reply content/about/faq.md]: Thanks! I was wondering what to link to here...

**Comment by agrabeli** [comment]
Thanks @hellais for the review! My latest commit includes edits based on your feedback. Are you sure it's ready to merge...?

**Comment by hellais** [comment]
I did another pass of reviewing this inside of the doc.

**Comment by agrabeli** [comment]
@hellais In my latest commit I addressed further feedback (from the Google doc version). If you think the FAQ is ready, please merge this PR.

**Pull request merged**